Bootcamp2015-AmazingRace/Views/MapPage.xaml.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using Windows.Devices.Geolocation;
+using Windows.Devices.Geolocation.Geofencing;
+using Windows.Services.Maps;
+using Windows.UI;
+using Windows.UI.Core;
+using Windows.UI.Xaml.Controls.Maps;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+// The Blank Page item template is documented at http://go.microsoft.com/fwlink/?LinkID=390556
+
+namespace Bootcamp2015.AmazingRace.Views
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class MapPage : Page
+    {
+        private Geolocator _locator;
+
+        public MapPage()
+        {
+            this.InitializeComponent();
+
+            _locator = new Geolocator() { ReportInterval = 1000 };
+            Pins = new ObservableCollection<PinViewModel>();
+
+            this.DataContext = this;
```

Comment:
This works in a pinch, but prefer that the View and ViewModel be implemented as separate classes. Caliburn makes it easy to wire these together using a file/folder naming convention.